CHANGELOG.md

```diff
@@ -63,8 +63,8 @@
 * **translator:** add history_translator ([#115](https://github.com/rime/librime/issues/115)) ([ae13354](https://github.com/rime/librime/commit/ae13354))
 
 
-
```

Comment:
The change log generator puts 3 new lines after a version level (`##`) paragraph, and 2 new lines between sections (`### Bug Fixes`, `### Features` etc.)
This can be reproduced by
```sh
npm install
npm run version
git diff --cached
```

Reply:
It doesn't specify how many lines are needed after the version title if immediately followed by contents.